coil-core/src/jvmCommonMain/kotlin/coil3/fetch/ByteBufferFetcher.kt

```diff
@@ -36,4 +37,23 @@ internal class ByteBufferFetcher(
             return ByteBufferFetcher(data, options)
         }
     }
+
+    class ByteBufferMetadata(val byteBuffer: ByteBuffer) : ImageSource.Metadata()
```

Comment:
Let's move this to the same file as `AssetMetadata` and mark it with `@ExperimentalCoilApi`.

Reply:
AssetMetadata is in androidMain, but ByteBufferFetcher is in jvmCommonMain

Reply:
Done